renku/core/session/session.py

```diff
@@ -103,6 +103,10 @@ def session_start(
 
     # set resource settings
     cpu_limit = cpu_request or get_value("interactive", "cpu_request")
+
+    if cpu_limit is not None:
+        cpu_limit = float(cpu_limit)
```

Comment:
We should catch a `ValueError` to avoid the `you've found a bug` message if `cpu_limit` is not a valid float str. I believe the passed-in `cpu_request` is checked by `validate_arguments` but the one that we read from the config is not checked.